packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -107,61 +89,74 @@ This should be added using the `aria-label` prop to each Item.
 
 ### Internationalization
 
-In order to internationalize an ActionGroup, the strings of all child items should be localized.
+In order to internationalize an ActionGroup, the content provided to all child items should be localized.
 
-## Events
+## Selection
+ActionGroup supports multiple selection modes. By default, selection is disabled however this can be changed using the `selectionMode` prop.
```

Comment:
```suggestion
ActionGroup supports multiple selection modes. By default, selection is disabled, however this can be changed using the `selectionMode` prop.
```